cql/ManageSpecialPopulation.cql

```diff
@@ -55,6 +55,26 @@ context Patient
 // RECOMMENDATIONS FOR SPECIAL POPULATIONS
 //------------------------------------------------------------------------------
 
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+// Special Populations Management of Patients With Histologic Cancer
+
+define RecommendationForPatientsWithHistologicCancer:
+  case
+    when (
+      Rare.HistologyInterpretedAsCancer
```

Comment:
Rare.HistologyInterpretedAsCancer only looks for whether the most recent biopsy was histologic cancer, not if there is ANY instance of Histologic Cancer as your comment mentions

Reply:
I think a test patient where histologic cancer was not the most recent biopsy would be helpful to show this.

Reply:
Changed to look for cancer in any prior biopsy. Added test case.

Reply:
Great! I like the new test case - shows we should be getting the gyn oncology referral recommendation if histologic cancer is present at all in the patient record, regardless of other biopsy results that may be present before or after.